### ISSUE TITLE
double-clicking the Open SQL Query button should make the query tab "sticky"

### ISSUE THREAD

**Issue by alvint** [TablePlus/TablePlus#579]
If you double-click on a table, it is opened in a "sticky" manner. That is, the tab that the table is opened in will be locked to the table's contents and clicking on another table will open it in a new tab. It would be nice if double-clicking on the Open SQL Query button worked in a similar manner. In order to make the query tab sticky, I have to open it and then remember to type something in it, which is inconvenient.

**Comment by alvint** [comment]
A separate issue prevents this from working properly here: If you resize the left panel at some point, the new panel size isn't used when viewing an SQL query. For example, at some point I widened the left panel because some of my table names are long. So the first time I click on the Open SQL Query button, the SQL view instantly displays and the left panel becomes narrow again. This causes the button to shift position, making it impossible to double-click.

I guess the real issue is that the SQL view doesn't use the correct left panel width--the panel always reverts back to a narrow width. If this were fixed then this should work as well. Should I open a new issue for that?
